pdf/pdf.py

```diff
@@ -38,17 +41,17 @@ class pdfXBlock(XBlock):
         default=_("http://tutorial.math.lamar.edu/pdf/Trig_Cheat_Sheet.pdf"),
         scope=Scope.content,
         help=_("The URL for your PDF."))
-    
+
```

Comment:
Please put those space changes in a separate commit, or just revert them.

Reply:
@OmarIthawi I removed the space changes for now.